docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -592,62 +593,106 @@ plugin libraries, composed of:
   host-side services for SGX quote verification.
 
 The OE SDK V0.10 release will also have enclave-side and host-side plugins
-for verification of SGX ECDSA reports generated by the legacy API function
-`oe_get_report()` and raw SGX ECDSA quotes generated by the Intel SGX SDK.
-These two evidence formats are identified by their UUIDs:
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT`: for SGX ECDSA report generated
-by `oe_get_report()`.
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`: for raw SGX ECDSA quote generated
-by the Intel SGX SDK.
+for verification of OE reports for SGX generated by the legacy API function
+`oe_get_report()` and raw SGX quotes generated by the Intel SGX SDK
+DCAP library. These two evidence formats are identified by two UUIDs:
+- `OE_FORMAT_UUID_SGX_REPORT`: for all OE reports generated
```

Comment:
Can we name this OE_FORMAT_UUID_OE_REPORT? SGX_REPORT implies local report.

Reply:
No, dropping SGX would imply it's TEE agnostic.

Reply:
Sure, agreed.

Reply:
> No, dropping SGX would imply it's TEE agnostic.

@dthaler the legacy API `oe_get_report()` only supports two report types for SGX: SGX report for local attestation, and SGX ECDSA quote for remote attestation.

How about calling it `OE_FORMAT_UUID_SGX_OE_REPORT`?

Reply:
OE is already in the beginning, so it looks bad to see it twice in the same define.

Reply:
It sounds like you're saying that oe_get_report is a misnomer since it also supports "SGX ECDSA quotes for remote attestation" which are not usually called "reports", is that correct?   (If so I guess it's a good thing we can obsolete this name eventually.)   What about something like OE_FORMAT_SGX_REPORT_OR_QUOTE?

Reply:
> It sounds like you're saying that oe_get_report is a misnomer since it also supports "SGX ECDSA quotes for remote attestation" which are not usually called "reports".

In SGX terminology, "SGX report" is for local attestation, and "SGX quote" is for remote attestation. QE converts reports to quotes. So it becomes confusing when `oe_get_report()` produces "report" / evidence for both local and remote attestation.

I think `OE_FORMAT_SGX_REPORT_OR_QUOTE` could be confused with `OE_FORMAT_SGX_QUOTE`.

How about calling it `OE_FORMAT_UUID_LEGACY_REPORT`?